src/sagemaker_xgboost_container/algorithm_mode/train.py

```diff
@@ -273,9 +278,9 @@ def train_job(train_cfg, train_dmatrix, val_dmatrix, train_val_dmatrix, model_di
                     checkpoint_dir=checkpoint_dir, early_stopping_data_name=early_stopping_data_name,
                     early_stopping_metric=early_stopping_metric, early_stopping_rounds=early_stopping_rounds,
                     save_model_on_termination=save_model_on_termination, is_master=is_master, fold=len(bst))
-                add_debugging(callbacks=callbacks, hyperparameters=train_cfg, train_dmatrix=cv_train_dmatrix,
-                              val_dmatrix=cv_val_dmatrix)
-
+                if XGB_VERSION < "1.3.0":
```

Comment:
`XGB_VERSION` is a hardcoded constant so this condition will never be satisfied, right? Should we just remove the call to `add_debugging` entirely instead?

Reply:
I was thinking about it,but if sagemaker debugger supports xgboost>1.3.0 in the future, this will cost little to update